pkgs/applications/networking/sniffers/wireshark/default.nix

```diff
@@ -24,9 +25,9 @@ stdenv.mkDerivation {
 
   buildInputs = [
     bison flex perl pkgconfig libpcap lua5 openssl libgcrypt gnutls
-    geoip libnl c-ares python libcap glib
+    geoip libnl c-ares python libcap glib zlib
   ] ++ optional withQt qt4
-    ++ optional withGtk gtk;
+    ++ (optionals withGtk [gtk pango cairo gdk_pixbuf]);
```

Comment:
Minor nit: the `(...)` brackets around that expression are redundant and can be removed.